src/aspire/estimation/covar2d.py

```diff
@@ -179,7 +185,7 @@ def identity(x):
             ctf_fb_k = ctf_fb[k]
             ctf_fb_k_t = blk_diag_transpose(ctf_fb_k)
             mean_coeff_k = blk_diag_apply(ctf_fb_k, mean_coeff)
-            covar_coeff_k = self._get_covar(coeff_k, mean_coeff_k)
+            covar_coeff_k = self._get_covar(coeff_k, mean_coeff_k.flatten())
```

Comment:
`flatten` is not needed.

Reply:
It is still needed unless we rewrite `blk_diag_apply`.  Keep the current version till the issue #111 is fixed.

Reply:
If `mean_coeff_k` is a vector, `flatten` is not needed.